### PR TITLE
removed lodash dep

### PR DESCRIPTION
**Pull request by jimmywarting**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -64,7 +63,7 @@ function Bootprint (withData, targetDir) {
 function loadFromFileOrHttp (fileOrUrlOrData) {
   // If this is not a string,
   // it is probably already the raw data.
-  if (!_.isString(fileOrUrlOrData)) {
+  if (!typeof fileOrUrlOrData === 'string') {
```
Comment (nknapp) [inline index.js]: I think this lint-error may be gone if the expression line is `typeof fileOrUrlOrData !== 'string'`

**Comment by nknapp** [comment]
If the tests pass and you squash the PR to a single commit, I will merge it.

**Comment by nknapp** [comment]
Closing due to inactivity. Please create a new PR if you still want this change

**Comment by nknapp** [comment]
I have included your changes into eb2d03a. Included in 1.0.3, thanks again.

**Comment by jimmywarting** [comment]
Github has a neat "squash and merge" action...

**Comment by nknapp** [comment]
It wasn't just that. The [change mentioned above](https://github.com/bootprint/bootprint/pull/22#pullrequestreview-15970918) have to be included in order to make the linter happy. It was just easier to copy these lines.

**Pull request closed without merging**